### ISSUE TITLE
Ikea INSPELNING providing power > 0W even if switch is turned off

### ISSUE THREAD

**Issue by nicola-spreafico** [Koenkk/zigbee2mqtt#24688]
### What happened?

This morning (more than 10h ago), I switched off the printer using the plug switch, as you can see here:
![image](https://github.com/user-attachments/assets/5dcf7906-4c89-4e35-8571-19033b692a04)

Despite being off, the power consumption is reading 3.2W which is not possibile considering the device is not powered.

I went to Z2M interface and found this:
![image](https://github.com/user-attachments/assets/afb4b5a0-61d4-4a27-82ba-959d0235fbb0)
Only after I pressed the refresh button I got the actual 0.0 value
![image](https://github.com/user-attachments/assets/2096fa30-64cf-45c5-a508-48b78f5c7cfc)

### What did you expect to happen?

If the switch is off, the read power should be 0.0

### How to reproduce it (minimal and precise)

Connect some appliance to smart plug, switch off the plug, see that power is not going to 0

### Zigbee2MQTT version

1.41.0 commit: cc31ceab

### Adapter firmware version

7.4.1 [GA]

### Adapter

smlight slzb-06m

### Setup

Dedicated LXC on Proxmox

**Comment by MaxRower** [comment]
You need to fine tune the reporting settings, so changes get reported with an acceptable delay.

**Comment by nicola-spreafico** [comment]
Hi @MaxRower ,
I understand you are referring to this view:
![image](https://github.com/user-attachments/assets/f8d7a2a3-9195-4e1d-86cb-c2e4333410a8)

As far I can see the default values are up to 60 (seconds I guess), but in my case after many hours the value was not updated, so I don't think this is the root cause.

**Comment by MaxRower** [comment]
I think those values are seconds, so it would be 65000 seconds, nearly a day?
Do you have the latest firmware on your plugs?

**Comment by MaxRower** [comment]
Additionally, you are missing the entry for the acPowerDivisor, so you will have wrong voltage values, probably?
Take a close look at that thread, there is a lot of discussion about that plug.
https://github.com/Koenkk/zigbee2mqtt/issues/23961

**Comment by nicola-spreafico** [comment]
Yes, I'm having a wrong voltage value, but I did not give much interest as I'm using this plug only for switch and power, voltage value are ignored in any case. I was my understanding that a fix is waited for the next release, but now from your message I understand I need to fix it by myself with the link you provided? In any case is not a priority for me.

I never touched the values inside the Reporting tab, how it works? All those values what they represent?

**Comment by nicola-spreafico** [comment]
Problem still there
![Image](https://github.com/user-attachments/assets/26af270b-e2d3-4f91-a20f-3849631c8966)

Switch is off (so device is entirely without current), but the plug notify 1W consumption (and it is like that since several hours).
Clicking the refresh button solve the issue, but it requires a manual action

**Comment by rpadafo** [comment]
Hello all, I have the same issue.

When I turn off the plug the power is the last power when the plug was on.
The only way to refresh the data is pushing the "Power refresh" button on zigbee2mqtt.

Any solution??

**Comment by rpadafo** [comment]
I think I have a solution... In "Reporting" section. Modify the "activepower" line, "max rep interval".
From 65000 to 600 (5 minutes for example).

![Image](https://github.com/user-attachments/assets/ceca0896-f062-4f95-9853-bab79f653ca8)
